src/patternfly/components/Wizard/examples/Wizard.md

```diff
@@ -250,6 +238,87 @@ import './Wizard.css'
       {{/drawer-panel}}
     {{/drawer-main}}
   {{/drawer}}
+  {{#> wizard-footer}}
+    {{#> button button--modifier="pf-m-primary" button--IsSubmit="true"}}
+      Next
+    {{/button}}
+    {{#> button button--modifier="pf-m-secondary"}}
+      Back
+    {{/button}}
+    {{#> wizard-footer-cancel}}
+      {{#> button button--modifier="pf-m-link"}}
+        Cancel
+      {{/button}}
+    {{/wizard-footer-cancel}}
+  {{/wizard-footer}}
+{{/wizard}}
+```
+
+### With drawer, collapsed
+```hbs isFullscreen
+{{#> wizard wizard--id="wizard-with-drawer-collapsed-example"}}
+  {{#> wizard-header}}
+    {{#> button button--modifier="pf-m-plain pf-c-wizard__close" button--attribute='aria-label="Close"'}}
+      <i class="fas fa-times" aria-hidden="true"></i>
+    {{/button}}
+    {{#> title title--modifier="pf-m-3xl pf-c-wizard__title"}}Wizard title{{/title}}
+    {{#> wizard-description}}
+      Here is where the description goes
+    {{/wizard-description}}
+  {{/wizard-header}}
+  {{#> wizard-toggle}}
+    {{#> wizard-toggle-list}}
+      {{#> wizard-toggle-list-item}}
+        {{#> wizard-toggle-num}}2{{/wizard-toggle-num}}
+        Configuration
+        {{> wizard-toggle-separator}}
+      {{/wizard-toggle-list-item}}
+      {{#> wizard-toggle-list-item}}
+        Substep B
+      {{/wizard-toggle-list-item}}
+    {{/wizard-toggle-list}}
+    {{> wizard-toggle-icon}}
+  {{/wizard-toggle}}
+  {{#> drawer drawer--id="wizard-with-drawer-example-drawer" drawer--IsInline="true"}}
+    {{#> drawer-main}}
+      {{#> drawer-content drawer-content--NoBody="true"}}
+        {{#> wizard-outer-wrap}}
+          {{#> wizard-inner-wrap}}
+            {{#> wizard-main}}
+              {{> wizard-template-nav}}
```

Comment:
It looks like nav got moved into main in some of these examples - is that intentional? In others it is a child of inner-wrap and comes before main. Seems odd that it's a child of main/main-body.